zebra/if_netlink.c

```diff
@@ -98,10 +99,26 @@ static void set_ifindex(struct interface *ifp, ifindex_t ifi_index,
 					EC_LIB_INTERFACE,
 					"interface rename detected on up interface: index %d was renamed from %s to %s, results are uncertain!",
 					ifi_index, oifp->name, ifp->name);
-			if_delete_update(oifp);
+			update = true;
 		}
 	}
 	if_set_index(ifp, ifi_index);
+
+	oifp = if_lookup_by_name_per_ns(zns, if_name);
```

Comment:
Here you rewrite the oifp pointer that was obtained by `if_lookup_by_index_per_ns` earlier.

Reply:
this change of code was useless. youre right.

Reply:
@idryzhov , is it ok for you ?